### ISSUE TITLE
Section links when clicking on a heading

### ISSUE THREAD

**Issue by robinrendle** [csswizardry/CSS-Guidelines#32]
Hey @csswizardry,

It would be neat if you could click on a heading and get a link to that particular section of the docs – sort of like articles on A List Apart.

**Comment by csswizardry** [comment]
A good idea! I shall have to see if there’s a GH Pages-supported Jekyll plugin for that :)

**Comment by nenadjelovac** [comment]
@csswizardry Until you find plugin, you can simply do something like this:

```
<h3 id="section-id"><a href="#section-id">SECTION NAME HERE</a></h3>
```

Then after a click you can manually copy from address bar. 

I needed this soooo many times now when I'm doing someone's code review and want to send them to cssguidelines ;) :D

**Comment by oltodo** [comment]
:+1:

Sometimes, I try to explain some things to some people based on your CSS guidelines. I'd like to give them a link for access directly to a section. But I'm forced to say something like that :

```
See http://cssguidelin.es/ at section "Naming UI Components"
```

**Comment by nenadjelovac** [comment]
@oltodo exactly my usecase :) I offered a quickfix above ;)

**Comment by jimmynotjim** [comment]
@oltodo the headings are linked from the Table of Contents, I just grab the url from there. A small :link: icon with a copy to clipboard script next to headings would be a nice addition.

**Comment by oltodo** [comment]
Thanks @nenadjelovac !

@jimmynotjim That's right for the table of contents, I had not thought about..

**Comment by nenadjelovac** [comment]
@jimmynotjim yup, you can use it like that. But why scroll (if you are at the middle of the page in the middle of reading) to the top if you don't have to? #firstWorldProblems :D
